__tests__/pages/resource/id/index.test.tsx

```diff
@@ -58,5 +59,20 @@ describe("resource ID render", () => {
         '"http://hl7.org/fhir/us/core/StructureDefinition/us-core-diagnosticreport-note"',
       ),
     ).toBe(true);
+
+    const deleteButton = screen.getByRole("button", {
+      name: "Delete",
+    }) as HTMLButtonElement;
+
+    // click the identified delete button
+    fireEvent.click(deleteButton);
+
+    //make sure a modal with the correct information shows up. Mantine modals are classified as dialogs.
+    const deleteModal = await screen.findByRole("dialog");
+    expect(deleteModal).toBeInTheDocument();
+    expect(within(deleteModal).getByText(/DiagnosticReport/)).toBeInTheDocument();
+    expect(within(deleteModal).getByText(/denom-EXM125-3/)).toBeInTheDocument();
+    expect(within(deleteModal).getByRole("button", { name: "Delete" })).toBeInTheDocument();
+    expect(within(deleteModal).getByRole("button", { name: "Cancel" })).toBeInTheDocument();
```

Comment:
I would put this in its own `it` statement. We try to keep test cases as small as possible so if they fail it is more narrowed down. I would also split up the other it statement in this file because it is testing three separate things...so in total maybe four separate it statements?

Reply:
Ok! I only found three places to break it up. I make sure the buttons display, then in a separate it statement I make sure the JSON resource displays, and in the last it statement I test that the modal appears on the delete button's click. Is that ok?

Reply:
Yes! I think just splitting it up even a little is very helpful and will help in the future.